base/redactor.go

```diff
@@ -90,8 +90,8 @@ func buildRedactorFuncSlice(valueOf reflect.Value, function func(interface{}) Re
 type RedactionLevel int
 
 const (
-	RedactNone RedactionLevel = iota
-	RedactPartial
+	RedactPartial RedactionLevel = iota
```

Comment:
As discussed add a RedactUnset value to differentiate between none/unset and set default value in setupAndValidateLogging